.github/workflows/release.yml

```diff
@@ -19,6 +19,8 @@ on:
 env:
   GO_VERSION: "1.23.3"
   REPOSITORY: flannel/flannel-cni-plugin
+  IMAGE_NAME: flannel-io/flannel
```

Comment:
should this one be `flannel-cni-plugin` ?

if not we should change here no?
https://github.com/flannel-io/flannel/pull/2177/files#diff-e12539ca526d07185b3d16099b32ff84530f80eca7616d4a17d7c7975531c531R169

Reply:
I don't know if it allows you to push the image to a different name from repo

Reply:
I fixed the name.